.github/workflows/pubsub_push_subscription.yml

```diff
@@ -55,11 +55,16 @@ jobs:
           emptyResponse=$(echo "[]")
           output=$(gcloud pubsub subscriptions list --filter=name=projects/${{ secrets.PROJECT_ID}}/subscriptions/${{ inputs.SERVICE }}-${{ inputs.ENV }}-${{ inputs.TOPIC }}-subscription --format="json")
```

Comment:
Can we move the `${{ inputs.SERVICE }}-${{ inputs.ENV }}-${{ inputs.TOPIC }}-subscription` in a variable since is the key element  for the check and the update